driver/razermouse_driver.c

```diff
@@ -753,6 +765,11 @@ static ssize_t razer_attr_read_is_charging(struct device *dev, struct device_att
     struct razer_report response_report = {0};
 
     switch(usb_dev->descriptor.idProduct) {
+    // Wireless mice that don't support is_charging
```

Comment:
So Synapse doesn't know whether the mouse is charging or not? I'm assuming it's displaying it somewhere in the UI, or not?

Reply:
No. The mouse has an AA battery, so it's not rechargeable.

Reply:
Oh haha. I'm torn between removing this and fixing frontends but I think in this case I think we can keep it like this. Please add the note about the AA batteries to the comment though, otherwise I might be confused in the future :wink:

Reply:
Added a comment on the `case` line.